src/__tests__/fixtures/schema/student.ts

```diff
@@ -0,0 +1,93 @@
+import { TigrisCollection } from "../../../decorators/tigris-collection";
+import { PrimaryKey } from "../../../decorators/tigris-primary-key";
+import { TigrisDataTypes, TigrisSchema } from "../../../types";
+import { Field } from "../../../decorators/tigris-field";
+
+/******************************************************************************
+ * `Student` class demonstrates a Tigris collection schema generated using
+ * decorators. Type of collection fields is inferred using Reflection APIs. This
+ * particular schema example:
+ * - infers the type of collection fields automatically using Reflection APIs
+ * - has multiple primary keys
+ *****************************************************************************/
+export const STUDENT_COLLECTION_NAME = "students";
+
+@TigrisCollection(STUDENT_COLLECTION_NAME)
+export class Student {
+	@PrimaryKey(TigrisDataTypes.INT64, { order: 1, autoGenerate: true })
+	id?: string;
+
+	@PrimaryKey(TigrisDataTypes.STRING, { order: 2 })
+	email: string;
+
+	@Field()
+	firstName!: string;
+
+	@Field()
+	lastName!: string;
+
+	@Field({ timestamp: "createdAt" })
+	createdAt?: Date;
+}
+
+/********************************** END **************************************/
+
+/******************************************************************************
+ * `InvalidStudent` class demonstrates a Tigris collection schema validation,
+ *  Schema is INVALID as it contains two primary keys but no order was specified
+ *  in decorator under PrimaryKeyOptions.
+ *****************************************************************************/
+export const INVALID_STUDENT_COLLECTION_NAME = "invalid_students";
+
+@TigrisCollection(INVALID_STUDENT_COLLECTION_NAME)
+export class InvalidStudent {
```

Comment:
nit: we use these examples for user references as well. Glad you added a negative test case. Would be great if we can remove this `InvalidStudent` schema from here and initialize it in the test case itself, like [here](https://github.com/tigrisdata/tigris-client-ts/blob/dc1c69d43bfac870e3d0c1754abccfd6bcad4bc0/src/__tests__/tigris.schema.spec.ts#L80-L93). Otherwise looks great.